src/OpenTK.Windowing.GraphicsLibraryFramework/Input/MouseState.cs

```diff
@@ -165,6 +161,16 @@ internal void Update()
             PreviousScroll = Scroll;
         }
 
+        /// <summary>
+        /// Gets a <see cref="bool" /> indicating whether this button is down.
```

Comment:
Let's make this really clear, because down vs pressed is always a point of confusion.
"A button is down as long as the button is physically pressed. This will be true every frame while it is held."